app/assets/javascripts/initializers/initializeSwipeGestures.js

```diff
@@ -1,61 +1,97 @@
-function initializeSwipeGestures(){
+function initializeSwipeGestures() {
   initializeSwipeGestures.called = true;
-  swipeState = "middle";
-  setTimeout(function(){
-
+  swipeState = 'middle';
+  setTimeout(function() {
     // window.onload=function(){
-    (function(d){
-    var
-    ce=function(e,n){var a=document.createEvent("CustomEvent");a.initCustomEvent(n,true,true,e.target);e.target.dispatchEvent(a);a=null;return false},
-    nm=true,sp={x:0,y:0},ep={x:0,y:0},
-    touch={
-      touchstart:function(e){sp={x:e.touches[0].pageX,y:e.touches[0].pageY,scrollY:window.scrollY}},
-      touchmove:function(e){nm=false;ep={x:e.touches[0].pageX,y:e.touches[0].pageY,scrollY:window.scrollY}},
-      touchend:function(e){
-        if(nm){ce(e,'fc')}
-        else {
-          var x=ep.x-sp.x,xr=Math.abs(x),y=ep.y-sp.y,yr=Math.abs(y),absScroll=Math.abs(sp.scrollY-ep.scrollY);
-          if(Math.max(xr,yr)>15){
-            var shouldScroll = (xr/2)>yr && absScroll < 5;
-            ce(e,(shouldScroll?(x<0?'swl':'swr'):(y<0?'swu':'swd')))
-          }
+    (function(d) {
+      var ce = function(e, n) {
+          var a = document.createEvent('CustomEvent');
+          a.initCustomEvent(n, true, true, e.target);
+          e.target.dispatchEvent(a);
+          a = null;
+          return false;
         };
-        nm=true
-      },
-      touchcancel:function(e){nm=false}
-    };
-    for(var a in touch){d.addEventListener(a,touch[a],false);}
+        var nm = true;
```

Comment:
I see prettier in action here, and it looks like this code was probably pulled from somewhere minified. It looks like it's just registering events, but if you have more insight, consider adding a comment about this.